src/main/java/by/custom/utilcalculator/domain/constants/steps/CurrentQuestion.java

```diff
@@ -0,0 +1,5 @@
+package by.custom.utilcalculator.domain.constants.steps;
+
+public enum CurrentQuestion {
```

Comment:
What if we need to store a previous question (step)? We have to create one more enum with name   `PreviousQuestion`? I recommend using more abstract name, just `Question` or `Step`.